src/hooks/usePictureRecordValidation.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export const usePictureRecordValidation = (pic: string | null | undefined) => {
+  const [isValid, setValidStatus] = useState(false);
+  const checkValidity = async () => {
+    try {
+      setValidStatus(false);
+      if (!pic) return;
+      await axios.get(pic);
+      setValidStatus(true);
+    } catch {}
+  };
+  useEffect(() => {
```

Comment:
Since this is a bit short and simple, is it preferred to set up the hook this way instead of using something like React Query?

Reply:
I'd say I just wanted to not overcomplicate and use something that already works. But yeah if start thinking about "is it worth using it", something like React Query is useless here because we don't really rely on cache, refresh, mutations, loading, and anything else in that specific case, so it will be just an overcomplication with zero positive effect